docs/run/advanced-runtime-options.mdx

```diff
@@ -105,53 +105,51 @@ suppression mechanisms. Level 0 only involves basic transpilation.
 See the Optimization level table in the
 [Error suppression topic](configure-error-suppression#setting-the-optimization-level) for further details.
 
+<Admonition>
+    When using Qiskit Runtime primitives, optimization levels 2 and 3 behave like level 1. If you want to use more advanced optimization, use the Qiskit transpiler locally, set `skip_transpilation=True`, and then pass the transpiled circuits to the primitives.  For instructions see the [Submitting user-transpiled circuits using primitives](https://learning.quantum-computing.ibm.com/tutorial/submitting-user-transpiled-circuits-using-primitives) tutorial.
+</Admonition>
+
 The optimization level option is a "first level option", and can be set as follows:
 
 ```python
 from qiskit_ibm_runtime import Estimator, Options
 
-options = Options(optimization_level=2)
+options = Options(optimization_level=1)
 
 # or..
 options = Options()
-options.optimization_level = 2
+options.optimization_level = 1
 
 estimator = Estimator(session=session, options=options)
 ```
 
-
-You might want to configure your transpilation strategy further, and for this, there are advanced transpilation
-options you can set up. These are "second-level options", and can be set as follows:
+The `skip_transpilation` option is an advanced transpilation option, set as follows:
```

Comment:
nit - I don't know why `skip_transpilation` is moved up. But this grouping becomes a bit odd:

1. `optimization_level` and `initial_layout`/ `routing_method` are all about tuning the transpiler. Whereas `skip_transpilation` is about skipping the transpiler. So it's odd to have this in the middle.
2. The words "second-level options" now appear later, even though `skip_transpilation` is also a second level option.

Reply:
I moved skip transpilation up because it's part of that note.  Maybe I can just add a link to that information.